BE/Cloud/KMS.Product.Ktm.Api/Startup.cs

```diff
@@ -49,6 +57,14 @@ public void ConfigureServices(IServiceCollection services)
             services.AddScoped<IKudoTypeRepository, KudoTypeRepository>();
             services.AddScoped<IKudoRepository, KudoRepository>();
             services.AddScoped<IEmployeeTeamRepository, EmployeeTeamRepository>();
+
+            // mapper
+            var mappingConfig = new MapperConfiguration(mc =>
+            {
+                mc.AddProfile(new AutoMapping());
+            });
+            IMapper mapper = mappingConfig.CreateMapper();
+            services.AddSingleton(mapper);
```

Comment:
Recommend to use services.AddAutoMapper() to add automapper profile
For the auto mapper profile, please use a more meaningful name (maybe: KudosProfile...)